### PR TITLE
Update container naming scheme

### PR DESCRIPTION
**Pull request by avisiedo**
- Read IPA_VERSION from the PARENT_IMG.
- For master branch tag 'latest' and the IPA_VERSION when no scheduled events.
- For master branch tag 'nightly' when a scheduled event.

### CODE REVIEW

**Comment by tiran** [comment]
We could look into adding custom labels to the FreeIPA container, e.g. ``LABEL org.freeipa.freeipa-server.version="version string"``. Labels can be set statically in the dockerfile or dynamically by the container builder. ``podman inspect`` can dump labels as JSON obejcts.

File: .github/workflows/containers.yaml [hunk 1]
```diff
@@ -35,9 +35,14 @@ jobs:
         run: |
           source ci/config/env
           echo ::set-output name=PARENT_IMG::${PARENT_IMG}
+          IPA_VERSION="$( docker run --rm --entrypoint "" "${PARENT_IMG}" rpm -q --qf '%{version}' freeipa-server )"
```
Comment (avisiedo) [inline .github/workflows/containers.yaml:38]: Try to retrieve this information from image metadata if it were available.

Comment (avisiedo) [reply .github/workflows/containers.yaml:38]: The PARENT_IMG does not have metadata information about the IPA version, so I leave it as it is.

**Comment by avisiedo** [comment]
> We could look into adding custom labels to the FreeIPA container, e.g. `LABEL org.freeipa.freeipa-server.version="version string"`. Labels can be set statically in the dockerfile or dynamically by the container builder. `podman inspect` can dump labels as JSON obejcts.

A new ticket will be for all the metadata (Dockerfile and Openshift resources).

**Review by tiran — APPROVED** [review]
``\o/``

**Pull request merged**